tests/filecheck/dialects/riscv_ssa/riscv_ssa_lowering.mlir

```diff
@@ -0,0 +1,14 @@
+// RUN: xdsl-opt -p lower-riscv-ssa %s | filecheck %s
+
+"builtin.module"() ({
+  %success = "riscv_ssa.syscall"() {"syscall_num" = 64 : i32}: () -> (!riscv.reg<s1>)
```

Comment:
:heart_eyes:

Reply:
because it's not directly convertible to risc-v assembly